delete_me_proxies_example/wqp_discrete_inventory.R

```diff
@@ -0,0 +1,655 @@
+# Before you run this script!
+# 1. This assumes that you have a folder called `0_inventory_exploration` in your current working directory
+# 2. It also assumes you have the SRS names crosswalk from WQP in your 0_inventory_exploration folder 
+#   called `public_srsnames_October_2020.csv`. You can also download the CSV from https://www.waterqualitydata.us/public_srsnames/
+# 3. You need to have the list of desired characteristics compiled by the PROXIES Data 3c group. These
+#   are stored in a file called `Characteristics for HABS, PFAS, METALS and COMMON.xlsx` and have 4 tabs
+
+stopifnot("0_inventory_exploration" %in% list.dirs(full.names=FALSE))
+stopifnot("public_srsnames_October_2020.csv" %in% list.files("0_inventory_exploration"))
+stopifnot("Characteristics for HABS, PFAS, METALS and COMMON.xlsx" %in% list.files("0_inventory_exploration"))
+
+library(readr)
+library(readxl)
+library(dplyr)
+library(dataRetrieval)
+library(xml2)
+library(feather)
+library(stringr)
+library(tidyr)
+library(writexl)
+
+# Define basins
+drb_huc4s <- "0204"
+irl_huc4s <- c("0712", "0713")
+uco_huc4s <- c("1401", "1402")
```

Comment:
Based on our scoping meeting, we'll be able to drop `irl_huc4s` and `uco_huc4s` , hooray!